api-spec/STAC-extensions.yaml

```diff
@@ -640,10 +640,13 @@ components:
     sortby:
       name: sortby
       in: query
-      description: Allows sorting results by the specified properties
+      description: >
+        An array of property names, prefixed by either "+" for ascending or "-"
+        for descending. If no prefix is provided, "+" is assumed.
       required: false
-      schema:
+      schema:ß
```

Comment:
There's a misplaced ß

Reply:
doh... fixed

Reply:
And I'm really wondering how you managed to accidentally add a ß (German umlaut) with a US keyboard :-D

Reply:
Umlaut?  Isn't that an Eszett? :D

At home I use a Windows keyboard on my Mac.   I'm in the habit of quickly pressing Windows-S after I change a file to save all.  Sometimes after I install updates or reboot the Alt and Windows keys magically swap.  Apparently when the keys swap, Windows-S creates an Eszett.  I have to go back into the Microsoft IntelliType software and restore defaults to get it back to normal.

tschüß

Reply:
Haha, nice! We often also refer to Eszett as Umlaut, because many non-German people are confused about that.